arches/app/models/migrations/11570_harden_editable_future_graphs.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 5.1.2 on 2024-11-13 18:17
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("models", "11726_join_tile_nodegroup"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="resource2resourceconstraint",
+            name="resourceclassfrom",
+            field=models.ForeignKey(
+                blank=True,
+                db_column="resourceclassfrom",
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="resxres_contstraint_classes_from",
+                to="models.node",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="resource2resourceconstraint",
+            name="resourceclassto",
+            field=models.ForeignKey(
+                blank=True,
+                db_column="resourceclassto",
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="resxres_contstraint_classes_to",
```

Comment:
It looks like there's an extra 't' in constraint

Reply:
wow, looks like this goes back to migration 1, and is a current issue in `models.py`. Mind if this is a followup?

Reply:
No problem - a followup sounds like the way to go.